src/types/featuresTypes.ts

```diff
@@ -3,4 +3,7 @@ export type PanelFeatureInterface = {
   panelIcon?: JSX.Element;
   content?: JSX.Element | null;
   minHeight?: number;
+  maxHeight?: number;
+  // noShrinking?
+  skipAutoResize?: boolean;
```

Comment:
should it be `noShrinking`?

Reply:
it passes to `useHeightResizer` with `noResize` name, so we can reuse this one